src/resources/js/tickets-registration-page.js

```diff
@@ -526,6 +578,68 @@ console.log(value);
 		}
 	);
 
+	/**
+	 * Handle AR submission.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	obj.document.on(
+		'click',
+		obj.selector.checkoutButton,
+		function( e ) {
+			e.preventDefault();
+			var $button      = $( this );
+			var $metaForm    = $( obj.selector.metaForm );
+			var $errorNotice = $( '.tribe-tickets-notice--error' );
+			var isValidForm  = obj.validateForm( $metaForm );
+
+			if ( ! isValidForm[ 0 ] ) {
+				$([document.documentElement, document.body]).animate(
+					{ scrollTop: $( '.tribe-tickets__registration' ).offset().top },
+					'slow'
+				);
+
+
+				$( '.tribe-tickets-notice--error__count' ).text( isValidForm[ 1 ] );
+				$errorNotice.show();
+
+				return false;
+			}
+
+			$errorNotice.hide();
+
+			// save meta
+			var params = {
+				provider: obj.providerId,
+				tickets : {},
+				meta    : obj.getMetaForSave(),
+				post_id : obj.postId,
+			};
+
+			$.ajax({
+				type: 'POST',
+				url: '/wp-json/tribe/tickets/v1/cart',
+				data: params,
+				success: function( response ) {
+					//redirect url
+					var url = response.checkout_url;
+					if ( undefined === url || ! url ) {
+						return false;
+					} else {
+					}
+
+					window.location.href = url;
+				},
+				fail: function( response ) {
+					// @TODO: add messaging on error?
```

Comment:
Handle error yet?

Reply:
Nope - I need some direction there I think. Do we want to display it to the user?

Reply:
Will be covered by https://central.tri.be/issues/134944